### PR TITLE
[Issue #91] Deep Insert only works with one level

### PR DESCRIPTION
**Pull request by ferjm**

### CODE REVIEW

File: src/models/db.js [hunk 1]
```diff
@@ -89,6 +89,14 @@ module.exports = config => {
     return [hasMany, belongsToMany].indexOf(type) > -1;
   };
 
+  db.getPlural = modelOptions => {
+    let modelName = modelOptions.name.plural;
+    if (modelName === 'FeaturesOfInterests') {
```
Comment (albertopq) [inline src/models/db.js]: What about ObservedProperty and ObservedProperties?

Comment (albertopq) [inline src/models/db.js]: Nevermind, I guess sequelize is handling correctly that plural

Comment (ferjm) [inline src/models/db.js]: Yes, it handle it properly in this case.

**Pull request merged**